CBLClient/Apps/CBLTestServer-Dotnet/TestServer/TestServer.csproj

```diff
@@ -16,7 +16,6 @@
     <PackageReference Include="JetBrains.Annotations" Version="11.1.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
     <PackageReference Include="Couchbase.Lite.Enterprise" Version="2.1.0-b0232" />
-    <PackageReference Include="Couchbase.Lite.Enterprise.Support.NetDesktop" Version="2.1.0-b0232" />
```

Comment:
It is not required to checkin this file? Can you remove this checkin? Checkin the updated file and then you can add it to gitignore to avoid checkins for this file

Reply:
It's required because jenkins job uses this file to build the app properly and because the appropriate packages were not added or extra packages were added, the jobs fails to build app.  This modification is required.